.github/workflows/update-dvc-sdk.yml

```diff
@@ -1,4 +1,7 @@
 name: Update DevCycle SDKs to Latest
+permissions:
+    contents: write
```

Comment:
does this need `contents: read` as well?

Reply:
write includes read